Tests/StreamChatTests/Controllers/MemberController/MemberController_Tests.swift

```diff
@@ -33,17 +33,13 @@ final class MemberController_Tests: XCTestCase {
     override func tearDown() {
         userId = nil
         cid = nil
+        client = nil
+        controller = nil
         controllerCallbackQueueID = nil
         
         env.memberUpdater?.cleanUp()
         env.memberListUpdater?.cleanUp()
-
-        AssertAsync {
-            Assert.canBeReleased(&controller)
-            Assert.canBeReleased(&client)
-            Assert.canBeReleased(&env)
-        }
```

Comment:
@nuno-vieira  Ill create task for this specific issue in our Testing Strategy epic. 
Calling `async` api in synchronously run `tearDown` is dangerous and adds up to the flakiness.
I see the need to leverage the right amount of automation in our tests to protect us against the memory leaks. Currently our SDK contains handful of them == this needs a bit of more research.

Reply:
Yes, this being async is not ideal for sure. Should we revert this change tho until we fix it properly in that new task? 🤔 Otherwise, it will be difficult to know where we need to put this

Reply:
@nuno-vieira I reversed the changes for now. Im going to merge the PR. its been around too long